deployments/scripts/deploy-k8s.sh

```
@@ -138,8 +172,22 @@ setup_provider() {
             RUN_HELM="aws_run_helm"
             RUN_HELM_WITH_VALUES="aws_run_helm_with_values"
             ;;
+        byo|microk8s)
+            # No cloud-specific kubectl/helm wrappers — use plain commands.
+            # The functions defined here mirror the azure/aws wrapper signatures
+            # so callers don't need to know which provider they're on.
+            byo_run_kubectl() { kubectl $*; }
```

Shellcheck (0.11.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)